__tests__/api/filtering_test.js

```diff
@@ -0,0 +1,62 @@
+let filtering = require('../../api/filtering');
+let mockData = require('../../__mocks__/mockData');
+let mockDataSingle = require('../../__mocks__/mockDataSingle');
+let mockDataEmpty = require('../../__mocks__/mockDataEmpty');
+// items is the actual array of events returned, see mockData for further details (meta data)
+mockData = mockData.mockData.items;
+mockDataSingle = mockDataSingle.mockData.items;
+mockDataEmpty = mockDataEmpty.mockData.items;
```

Comment:
You could have a single mock object (`mockData `) and change its content ad hoc instead of having 3 different mocks

Reply:
Yup, definitely sounds nicer. Just changed now and pushed.